pkg/option/config.go

```diff
@@ -2767,6 +2771,15 @@ func (c *DaemonConfig) checkMapSizeLimits() error {
 	return nil
 }
 
+func (c *DaemonConfig) checkIPv4NativeRoutingCIDR() error {
+	if c.IPv4NativeRoutingCIDR() == nil && c.Masquerade && c.Tunnel == TunnelDisabled && c.IPAMMode() != IPAMENI {
+		return fmt.Errorf("native routing cidr must be configured with option --%s in combination with --%s --%s=%s --%s=%s",
+			Masquerade, IPv4NativeRoutingCIDR, TunnelName, c.Tunnel, IPAM, c.IPAMMode())
```

Comment:
I think the argument order is still off here. I think you want `IPv4NativeRoutingCIDR` for the first `%s`

Reply:
You are of course right. Fixed.